src/checks.cc

```diff
@@ -403,8 +404,33 @@ LinterResult CheckImports(absl::string_view sql, const LinterOptions &option) {
 
 LinterResult CheckExpressionParantheses(absl::string_view sql,
                                         const LinterOptions &option) {
-  LinterResult result;
-  return result;
+  return ASTNodeRule([](const ASTNode *node, const absl::string_view &sql,
+                        const LinterOptions &option) -> LinterResult {
+           LinterResult result;
+           if (node->node_kind() == AST_OR_EXPR ||
+               node->node_kind() == AST_AND_EXPR) {
+             if (node->parent() == nullptr) return result;
+             const ASTNode *parent = node->parent();
+
+             if (parent->node_kind() == AST_OR_EXPR ||
+                 parent->node_kind() == AST_AND_EXPR) {
+               if (parent->node_kind() == node->node_kind()) return result;
+               int pos = node->GetParseLocationRange().start().GetByteOffset();
+               int start = pos - 1;
+               int end = node->GetParseLocationRange().end().GetByteOffset();
+
+               if (IgnoreBackwardSpaces(sql, &start) ||
+                   IgnoreForwardSpaces(sql, &end) || sql[start] != '(' ||
+                   sql[end] != ')')
+                 if (option.IsActive(ErrorCode::kExpressionParanteses, pos))
+                   result.Add(ErrorCode::kExpressionParanteses, sql, pos,
+                              "Use parantheses between consequtive AND and OR "
+                              "statements.");
```

Comment:
s/statements/operators/

Reply:
Made it "operators" (and there shouldn't be '.' at the end, since there will be '[<>]' )